pkg/pillar/cmd/volumemgr/handlepvcdiskmetrics.go

```
@@ -0,0 +1,66 @@
+// Copyright (c) 2025 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build kubevirt
+
+package volumemgr
+
+import (
+	"github.com/lf-edge/eve/pkg/pillar/kubeapi"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+)
+
+// createOrUpdatePvcDiskMetrics creates or updates metrics for all kubevirt PVCs
+// PVC mknod will match one of existing sdX devices, create copies for convenience
+func createOrUpdatePvcDiskMetrics(ctx *volumemgrContext) {
+	log.Functionf("createOrUpdatePvcDiskMetrics")
+	var diskMetricList []*types.DiskMetric
+
+	sdMajMinToNameMap, _, _ := kubeapi.SCSIGetMajMinMaps()
+	_, pvNameToMajMin, _ := kubeapi.LonghornGetMajorMinorMaps()
+	_, pvcToPvMap, _ := kubeapi.PvPvcMaps()
```

Please, don't ignore errors in the functions above...

---

Thanks for catching that, I'll fix it